doc/src/reference/integration_status.md

```diff
@@ -0,0 +1,75 @@
+# Integration Status
+
+This section gives a status of libmicrovmi's integration into
```

Comment:
```suggestion
This section gives a status overview of libmicrovmi's integration into
```
I don't understand the last part of this sentence, could you elaborate perhaps?

Reply:
It should be like this:
"This section gives a status of libmicrovmi's integration into other applications and libraries based on _virtual machine introspection_."